plugins/modules/system/listen_ports_facts.py

```diff
@@ -72,11 +84,21 @@
       returned: if TCP servers were found
       type: list
       contains:
-        address:
+        local_address:
           description: The address the server is listening on.
           returned: always
           type: str
           sample: "0.0.0.0"
+        foreign_address:
+          description: The address of the remote end of the socket.
+          returned: if ``include_non_listening`` is set
+          type: str
+          sample: "10.80.0.1"
```

Comment:
```suggestion
          sample: "10.80.0.1"
          version_added: 5.4.0
```
Same for the other new return values.

Reply:
added for return values and the paramter